org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/DefaultIdeModule.xtend

```diff
@@ -20,4 +28,17 @@ class DefaultIdeModule extends AbstractGenericModule {
 		binder.bind(ExecutorService).toProvider(ExecutorServiceProvider)
 	}
 	
+	def void configureIResourceDescriptionsLiveScope(Binder binder) {
+		binder.bind(IResourceDescriptions)
+			.annotatedWith(Names.named(ResourceDescriptionsProvider.LIVE_SCOPE))
+			.to(LiveShadowedChunkedResourceDescriptions);
+	}
+	
+	def Class<? extends IContainer.Manager> bindIContainer$Manager() {
+		ProjectDescriptionBasedContainerManager
+	}
+	
+	def Class<? extends IProjectConfigProvider> bindIProjectConfigProvider() {
+		ProjectConfigProvider
```

Comment:
why do we use this in XbaseRuntimeModule but not DefaultRuntimeModule?

Reply:
No I means the binding is inconsistent done for xbase and non xbase plus defaultide 

So I wonder if it should go to defaultruntime

Reply:
You mean in DefautIdeRuntimeModule but not DefaultRuntimeModule?

We could. But this will definitely affect existing clients and I am not sure about the impact.

Reply:
No I mean it’s inconsistent

Reply:
Why is binding in xbaseruntimemodule